climetlab_s2s_ai_challenge/extra.py

```diff
@@ -53,16 +53,35 @@ def forecast_like_observations(forecast, obs_time):
             `valid_time` coordinate
 
     Example:
+        >>> import climetlab as cml
+        >>> forecast = cml.load_dataset('s2s-ai-challenge-training-input',
+        ...     date=20100107, origin='ncep', parameter='tp',
+        ...     format='netcdf').to_xarray()
+        >>> obs_lead_forecast_time = cml.load_dataset('s2s-ai-challenge-observations',
+        ...     parameter=['pr', 't2m']).to_xarray(match=forecast)
```

Comment:
Instead of "match=", I would use "coords_like=" or "shape_like" or "like" in the argument of to_xarray(). 
(see https://numpy.org/doc/stable/reference/generated/numpy.full_like.html). But this is fine.